drivers/vulkan/SCsub

```diff
@@ -5,14 +5,12 @@ Import('env')
 env.add_source_files(env.drivers_sources, "*.cpp")
 
 if env['builtin_vulkan']:
-    # Use bundled Vulkan headers
-    thirdparty_dir = "#thirdparty/vulkan"
-    env.Prepend(CPPPATH=[thirdparty_dir, thirdparty_dir + "/include", thirdparty_dir + "/loader"])
-
     # Build Vulkan loader library
     env_thirdparty = env.Clone()
     env_thirdparty.disable_warnings()
 
+    use_vulkan_loader = True
```

Comment:
When false, this basically disables the whole `if env['builtin_vulkan']`, i.e. it's not using the builtin vulkan code. So instead of doing it this way, simply do:
```
if env['builtin_vulkan'] and not env['platform'] == "android":
...
elif env['platform'] == "android":
...
```